examples/httpd/convox.yml

```diff
@@ -2,3 +2,11 @@ services:
   web:
     image: httpd
     port: 80
+  worker:
+    build: ./worker
+    scale: 0
+timers:
+  cleanup:
+    command: ./cmd.sh
+    schedule: "*/1 * * * ?"
+    service: worker
```

Comment:
Can you link the timer to the `web` service? So you won't need to create another service, keeping it simpler.

Reply:
hey, @Twsouza, thank for your comments. Could you have a look again, please?